File: bindings/python/pydrake/test/scratch.m
%%
x = py.numpy.random.random(([4, 4]));
% % Printing this throws an error
% y = py.numpy.matlib.matrix([1, 2, 3]);

%%
pyscratch = pyimport('scratch');
py.reload(pyscratch);
% Call "clear classes" if you get instance method type mismatches.
% http://www.mathworks.com/help/matlab/matlab_external/call-modified-python-module.html

a = pyscratch.stuff_py()
b = pyscratch.stuff_py(pyargs('is_int', true))
c = pyscratch.stuff_numpy()

% % Will throw an error
% m.stuff_matlab()

File: bindings/python/pydrake/test/scratch.py
def stuff_py(is_int=False):
    if is_int:
        return 1
    else:
        return 1.

def stuff_numpy():
    import numpy as np
    import numpy.matlib as nm
    return nm.matrix([1, 2, 3])

def stuff_matlab():
    # Does not work
    import matlab
    return matlab.double(1.)
